infra/main.bicep

```diff
@@ -758,12 +746,11 @@ module server 'server.bicep' = {
     cosmosDbContainer: cosmosDbContainerName
     cosmosDbUserContainer: cosmosDbUserContainerName
     cosmosDbOAuthContainer: cosmosDbOAuthContainerName
-    applicationInsightsConnectionString: useAppInsights ? applicationInsights!.outputs.connectionString : ''
-    openTelemetryPlatform: openTelemetryPlatform
+    applicationInsightsConnectionString: useMonitoring ? applicationInsights!.outputs.connectionString : ''
     exists: serverExists
     // Keycloak authentication configuration (only when enabled)
-    keycloakRealmUrl: useKeycloak ? '${keycloak!.outputs.uri}/realms/${keycloakRealmName}' : ''
-    keycloakTokenIssuer: useKeycloak ? '${keycloakMcpServerBaseUrl}/realms/${keycloakRealmName}' : ''
+    keycloakRealmUrl: useKeycloak ? '${keycloak!.outputs.uri}/auth/realms/${keycloakRealmName}' : ''
```

Comment:
These are both referenced multiple times (here and in outputs) so might want to make more vars below line 730?